src/main/java/frc/robot/subsystems/superstructure/arm/ArmIOReal.java

```diff
@@ -93,7 +101,9 @@ public void config() {
         leftTalonFXConfiguration.TorqueCurrent.PeakReverseTorqueCurrent = -80;
         leftTalonFXConfiguration.CurrentLimits.StatorCurrentLimit = 60;
         leftTalonFXConfiguration.CurrentLimits.StatorCurrentLimitEnable = true;
-        leftTalonFXConfiguration.Feedback.SensorToMechanismRatio = armConstants.pivotGearing();
+        leftTalonFXConfiguration.Feedback.FeedbackSensorSource = FeedbackSensorSourceValue.FusedCANcoder;
```

Comment:
When using `FusedCANcoder`, the position data needs to be accessed from the motor that the cancoder data was fused with, not from the cancoder itself (hence, fused, because the encoder fuses its measurement with that of the motor).